hnn_core/gui/gui.py

```diff
@@ -229,11 +234,35 @@ def __init__(self, theme_color="#8A2BE2",
                                             description='Cores:',
                                             disabled=False)
         self.load_data_button = FileUpload(
-            accept='.txt', multiple=False,
+            accept='.txt,.csv', multiple=False,
             style={'button_color': self.layout['theme_color']},
             description='Load data',
             button_style='success')
 
+        b64 = base64.b64encode("".encode())
+        payload = b64.decode()
+        # Initialliting HTML code for download button
+        self.html_download_button = '''
+        <a download="{filename}" href="data:text/csv;base64,{payload}"
+          download>
+        <button style="background:{color_theme}; height:{btn_height}"
+        class=" jupyter-button
+           mod-warning" {is_disabled} >Save Simulation</button>
+        </a>
+        '''
+        # Create widget wrapper
+        self.save_simuation_button = (
+            HTML(self.html_download_button.
+                 format(payload=payload,
+                        filename={""},
+                        is_disabled="disabled",
+                        btn_height=self.layout['run_btn'].height,
+                        color_theme=self.layout['theme_color'])))
```

Comment:
This is a bit busy for the init. Can we put this into a hidden method to make the init cleaner? I don't think we need to save the html as an attribute. It can just be defined in the scope of the method. 
```suggestion
        self.save_simulation_button = self._init_html_download_button()
```

Reply:
I moved the init logic to a private function. I keep the html as an attribute due it needs to be updated later in the workflow